app/generated_resources.grd

```diff
@@ -9608,13 +9816,40 @@ Please help our engineers fix this problem. Tell us what happened right before y
 
     <!-- User happiness tracking survey UI -->
     <if expr="not is_android">
-      <message name="IDS_HATS_BUBBLE_OK_LABEL" translateable="false" desc="Button label on Happiness Tracking Survey's invitation banner, clicking the button gives the user's consent on taking the survey.">
-        Take Survey
-      </message>
+      <if expr="use_titlecase">
+        <message name="IDS_HATS_BUBBLE_OK_LABEL" translateable="false" desc="Button label on Happiness Tracking Survey's invitation banner, clicking the button gives the user's consent on taking the survey.">
+          Take Survey
+        </message>
+      </if>
+      <if expr="not use_titlecase">
+        <message name="IDS_HATS_BUBBLE_OK_LABEL" translateable="false" desc="Button label on Happiness Tracking Survey's invitation banner, clicking the button gives the user's consent on taking the survey.">
+          Take survey
+        </message>
+      </if>
       <message name="IDS_HATS_BUBBLE_TEXT" translateable="false" desc="The text in Happiness Tracking Survey's invitation banner, it explains why users should take our survey.">
         Your feedback is important to us.
       </message>
     </if>
+    <message name="IDS_NOTIFICATION_DEFAULT_HELPFUL_BUTTON_TEXT" desc="Text body for the helpful button in notifications sent from notification scheduler.">
+      Helpful
+    </message>
+    <message name="IDS_NOTIFICATION_DEFAULT_UNHELPFUL_BUTTON_TEXT" desc="Text body for the unhelpful button in notifications sent from notification scheduler.">
+      Not Helpful
+    </message>
+
+    <!-- SMS strings -->
+    <message name="IDS_SMS_INFOBAR_TITLE" desc="Title shown when Brave is waiting for an SMS on the user's behalf">
```

Comment:
Looks like SMSReceiver is enabled by default in C78: https://cs.chromium.org/chromium/src/content/public/common/content_features.cc?l=523. We should probably disable it till we have sufficient privacy protections in place. cc: @tomlowenthal @snyderp

Reply:
oh wow - good find!

Reply:
Looks like this happened in C76: https://storage.googleapis.com/chromium-find-releases-static/31e.html#31ee01f

Reply:
Addressed by cd673346449c29f47c4b3b8f51d5b02f923dde52